### PR TITLE
Convert style of dynamic toast messages to be like Bootstrap Alerts

### PR DESCRIPTION
**Pull request by kriscooke**
- Refactors the styling of the react-toastify toaster used in BaseMutation to appear similar to Bootstrap Alerts, which we use for static user messaging
  - top center placement
- Converts the `/admin/products-benchmarks` page to use the BaseMutation success message instead of [yet another](https://react-bootstrap.netlify.app/components/toasts/) different Toast

[GGIRCS-1639](https://youtrack.button.is/issue/GGIRCS-1639)

Screen Shot 2021-05-21 at 12 29 48 PM
Screen Shot 2021-05-21 at 12 58 01 PM

### CODE REVIEW

File: app/containers/Products/ProductCreatorContainer.tsx [hunk 1]
```diff
@@ -45,13 +43,14 @@ export const ProductCreator: React.FunctionComponent<Props> = ({
             e.formData.subtractGeneratedHeatEmissions,
           requiresProductAmount: e.formData.requiresProductAmount
         }
+      },
+      messages: {
+        success: 'Product created successfully.'
```
Comment (kriscooke) [inline app/containers/Products/ProductCreatorContainer.tsx:48]: Uses the BaseMutation toaster instead now

File: app/mutations/BaseMutation.ts [hunk 2]
```diff
@@ -75,9 +75,9 @@ export default class BaseMutation<T extends BaseMutationType = never> {
             reject(error);
             if (failureMessage) {
               toast(failureMessage, {
-                className: 'mutation-toast Toastify__toast--error',
+                className: 'toastalert-error',
```
Comment (kriscooke) [inline app/mutations/BaseMutation.ts:78]: We didn't need all these classes, and the `Toastify__toast` etc. ones are added automatically by react-toastify.

File: app/static/base.css [hunk 3]
```diff
@@ -47,15 +47,3 @@
...
-}
-.mutation-toast .Toastify__toast-body {
-  color: white;
-}
-.mutation-toast.Toastify__toast--success {
-  background: #49b34c;
-}
-.mutation-toast .Toastify__progress-bar--default {
-  background: #add059;
-}
```
Comment (kriscooke) [inline app/static/base.css:61]: Moved Toaster style overrides into the Toaster helper component.

**Review by matthieu-foucault — APPROVED** [review]
:shipit:

**Pull request merged**